samples/meetings-content-bubble/csharp/TargetedNotifications/Pages/ContentBubble.cshtml.cs

```diff
@@ -1,11 +1,12 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
+// <copyright file="ContentBubble.cshtml.cs" company="Microsoft">
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// </copyright>
+
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace Content_Bubble_Bot.Pages
+namespace TargetedInNotificationMeetingBot.Pages
 {
     public class ContentBubbleModel : PageModel
```

Comment:
Is this `contentBubbleModel` being used for in-meeting notification scenario (the existing feature) or targeted in-meeting notification scenario (the new feature)? 

Depending on which feature this file is for, can we update the name accordingly? 
ex: If this model class is for the in-meeting notification, we can update the name to be InMeetingNotificationModel. 

If this model classi s for the targeted in-meeting notification, we can update the name to be TargetedInMeetingNotificationModel or TargetedMeetingNotification Model. 

Same question for ContentBubble.cshtml file name as well :)

More about the features:
In-meeting notification (previously called, content bubble): https://review.learn.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/meeting-apps-apis?branch=pr-en-us-7615&tabs=dotnet#send-an-in-meeting-notification

Targeted Meeting notification: https://review.learn.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/meeting-apps-apis?branch=pr-en-us-7615&tabs=dotnet#targeted-meeting-notification-api

Reply:
Updated this based on page model